src/converters/BindingConverters.ts

```diff
@@ -22,16 +22,56 @@ export function getBindingDefinitions(info: FunctionInfo): BindingDefinition[] {
 }
 
 export function getNormalizedBindingData(request: rpc.IInvocationRequest): Dict<any> {
+
+  // Repopulate triggerMetaData if http.
+  repopulateHttp(request);
```

Comment:
I'm a little torn on this, I know request originally had triggerMetadata with http, but I'm weary of this method just modifying request. And if that's what it's intended to do, I would move it to be called closer to where request is first received.

Reply:
Actually I'm a little torn. I think it's cleaner to keep everything here since it's everything that creates "bindingData" from "triggerMetadata". Still a little weary of changing the object directly though at an arbitrary point

Reply:
I think I'm inclined towards what you initially suggested. Taking a second look, it is a little weird to modify the input parameter not just here but with any function invocation in general when the function's intention is just to create "bindingData" from whatever is provided as input. Let's chat tomorrow about where best to place it within the WorkerChannel so that it's reflected throughout.